### DISCUSSION TITLE
unhelpful error message when setting up database

### DISCUSSION THREAD

**Discussion by Devner-tour-guides-collective** [prisma/prisma#13510] [Q&A] [answered]
Not sure what to do about this, the error message provides absolutely no useful information. Not looking for help here, it would take too long to explain the problem, it would be faster just to start over. Just posting this to let you guys know that you should probably give more specific error messages.

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/106288660/170511152-a3373d7b-e6f1-41f6-a4ca-8e13697041d8.png">

**Comment by nurul3101** [comment, marked as answer]
Hey @Devner-tour-guides-collective 👋 

Thank you for reporting this issue, this issue has been introduced recently and our team is working on it to get it fixed.
I'll pass on this feedback though.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
